pom.xml

```diff
@@ -66,35 +62,19 @@ under the License.
     </properties>
 
     <modules>
-        <module>flink-connector-bigquery</module>
-        <module>flink-sql-connector-bigquery</module>
-        <module>flink-connector-bigquery-examples</module>
-        <module>flink-connector-bigquery-integration-test</module>
+        <module>flink-connector-bigquery-common</module>
     </modules>
 
     <dependencies>
-        <dependency>
-            <groupId>com.google.cloud</groupId>
-            <artifactId>google-cloud-bigquerystorage</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.flink</groupId>
-            <artifactId>flink-avro</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.flink</groupId>
-            <artifactId>flink-metrics-dropwizard</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.flink</groupId>
-            <artifactId>flink-shaded-force-shading</artifactId>
-        </dependency>
-
         <!-- Root dependencies for all projects -->
         <dependency>
             <groupId>dev.failsafe</groupId>
             <artifactId>failsafe</artifactId>
         </dependency>
+        <dependency>
```

Comment:
As this is used only in 1.17, move this dependency to the 1.17 module.

Reply:
ACK, Fixed.